boxsdk/config.py

```diff
@@ -9,3 +9,9 @@ class API(object):
     UPLOAD_URL = 'https://upload.box.com/api/2.0'
     OAUTH2_API_URL = 'https://api.box.com/oauth2'  # <https://developers.box.com/docs/#oauth-2>
     OAUTH2_AUTHORIZE_URL = 'https://app.box.com/api/oauth2/authorize'  # <https://developers.box.com/docs/#oauth-2-authorize>
+
+
+class Version(object):
```

Comment:
Would prefer not to add more global variables. As it is, I have an in-progress change to try removing our reliance on the global ``API`` configs.

Can we instead get rid of ``Version.VERSION`` (it isn't a config and it isn't necessary; we can just define ``boxsdk.version.__version__`` as proposed below), and move ``USER_AGENT_STRING`` into a class attribute on ``BoxSession``?